lib/google_apis.ex

```diff
@@ -34,9 +34,9 @@ defmodule GoogleApis do
          {:ok, %{"items" => apis}}           <- Poison.decode(body)
     do
       json = apis
-      |> Enum.filter(&GoogleApis.DirectoryItem.preferred?/1)
-      |> Enum.map(fn (api) -> Map.take(api, ["name", "discoveryRestUrl", "version"]) end)
-      |> Poison.encode!(pretty: true)
+        |> Enum.filter(&GoogleApis.DirectoryItem.preferred?/1)
```

Comment:
Let's move the `apis` to the next line to match the assigning of a multi-line expression.

Reply:
@chingor13 Done!